idea/src/org/jetbrains/kotlin/idea/actions/NewKotlinFileAction.java

```diff
@@ -55,8 +55,8 @@ protected void postProcess(PsiFile createdElement, String templateName, Map<Stri
     @Override
     protected void buildDialog(Project project, PsiDirectory directory, CreateFileFromTemplateDialog.Builder builder) {
         builder
-                .setTitle(JetBundle.message("new.kotlin.file.action"))
-                .addKind("Kotlin file", JetFileType.INSTANCE.getIcon(), "Kotlin File")
+                .setTitle("New Kotlin File/Class")
```

Comment:
Other stuff under the `New...` menu doesn't start with "New", this probably shouldn't start as well